mongodbatlas/resource_mongodbatlas_auditing.go

```diff
@@ -30,6 +30,7 @@ func resourceMongoDBAtlasAuditing() *schema.Resource {
 			"project_id": {
 				Type:     schema.TypeString,
 				Required: true,
+				ForceNew: true,
```

Comment:
Could you update the documentation to point out this new behaviour?

Reply:
Added Note to parameter